batcher/client/src/main.rs

```diff
@@ -75,27 +89,37 @@ async fn main() {
         println!("Warning: no VM Program Code File, continuing with no VM Program Code");
     }
 
-    let task = VerificationData {
+    // Dummy address for testing.
+    let addr_str = "0x66f9664f97F2b50F62D13eA064982f936dE76657";
+    let proof_generator_addr: Address = Address::parse_checksummed(addr_str, None).unwrap();
+
+    let verification_data = VerificationData {
         proving_system,
         proof,
-        public_input,
+        pub_input,
         verification_key,
         vm_program_code,
+        proof_generator_addr,
```

Comment:
It's needed for the full read flow. I can show to you tomorrow. It's to avoid someone watching the blockchain to frontrun the submission of your proofs.